changelog.txt

```diff
@@ -34,6 +34,7 @@ that repo.
 - `devel/query`: can now properly index vectors in the --table argument
 -@ `forbid`: fix detection of unreachable items for items in containers
 -@ `unforbid`: fix detection of unreachable items for items in containers
+-@ `gui/create-item`: adding corpse spawning broke spawning generic corpsepieces, but has now been fixed.
```

Comment:
50-07-alpha2 has already been released these entries should go in the "Future" section above.

Reply:
oop